src/context.c

```diff
@@ -72,6 +74,48 @@ static gboolean launch_and_wait_variables_handler(gchar *handler_name, GHashTabl
 	return res;
 }
 
+/*
+ * Mounts slot (if necessary), tries to read status file and saves information
+ * in RaucSlot struct.
+ */
+static void set_slot_status(RaucSlot *slot) {
```

Comment:
naming it `set_` might be a bit confusing here, what about `obtain_` or something similar?